v2/session/advanced-customizations/localstorage-example/about.mdx

```diff
@@ -0,0 +1,36 @@
+---
+id: about
+title: About
+hide_title: true
+---
+
+<!-- COPY DOCS -->
```

Comment:
You can use 
```
<!-- COPY DOCS -->
<!-- ./session/advanced-customizations/localstorage-example/about.mdx -->
```
here. it points to itself, but it will work.